### PR TITLE
fix: cursor position jumping to end.

### PR DESCRIPTION
**Pull request by ted-marozzi**
This fixes an issues observed in chrome where if you type enter then try to type anywhere that's not the end the cursor will jump to the end.

This is because the next props.html will have a `<br />` but the el.innerHTML will have been transformed to a regular `<br>` by the browser. The fix is to normalize the `<br />` to `<br>` recognizing they are equivalent.
Here is how I demonstrated the issue.
```
    if (normalizeHtml(nextProps.html) !== normalizeHtml(el.innerHTML)) {
      console.log(
        "left:",
        normalizeHtml(nextProps.html),
        "right:",
        normalizeHtml(el.innerHTML)
      );
      return true;
    }
```
I then hit enter and observed the following logs.
```
left: <div><br /></div><div><br /></div> right: <div><br></div><div><br></div>
```

As we can see the contenteditable thinks it needs to be updated so we lose cursor position but really we shouldn't update as the html is actually equivalent, hence my change maps both the br tags to the default html br tag.

### CODE REVIEW

**Comment by ted-marozzi** [comment]
@lovasoa Hey, let me know if I am missing anything or have anything I should change :)

**Comment by ted-marozzi** [comment]
Thanks @lovasoa for letting the CI run, I added --openssl-legacy-provider to your test code as node 18 got stricter with ssl. If you run npm audit fix --force on your test package it should remove the need for this. I didn't do that as I am not familiar with the test code and couldn't run locally. Also its not a security issue as it is test code and this is just preserving the node 16 behavior. Do you mind letting it run again? Thanks.

**Comment by ted-marozzi** [comment]
Seems like a typescript error in your test setup. Im not familiar with this test framework so probably won't dig into how to solve it as it's unrelated to my code change. For now I will just use a forked version of the package but if you do manage to unblock this error I would love to help get it merged and contribute back.

Cheers.

**Comment by ted-marozzi** [comment]
Okay I ended up getting it passing locally, mind giving it a rerun @lovasoa ?
Screenshot 2023-02-22 at 2 36 31 pm
Thanks

File: tests/tsconfig.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "preserve"
+    "jsx": "react-jsx",
+    "noFallthroughCasesInSwitch": true
   },
```
Comment (ted-marozzi) [inline tests/tsconfig.json:22]: These were automatic

**Comment by ted-marozzi** [comment]
@lovasoa not sure if you have been getting these messages or your CI runs automatically. If you are getting these messages is there anything I can do to help get this merged? Also at this stage is this package still maintained?

Thanks.

**Comment by lovasoa** [comment]
Yes, sorry for my lack of reactiveness. I'll merge !

**Comment by ted-marozzi** [comment]
No need to say sorry, thanks for you free package and for merging!

**Comment by ted-marozzi** [comment]
I was thinking about making a Hooks version of this, do you think that would work? If so I'd be happy for it to be a pr to this library instead of a new package if you think it would be a good idea.

**Comment by lovasoa** [comment]
Yes, that would make it easier for me, thank you ! Your PR would be welcome.

**Pull request merged**